package.json

```diff
@@ -10,17 +10,18 @@
     "lint:fix": "yarn lint --fix",
     "lint": "eslint packages --report-unused-disable-directives",
     "prepare": "lerna run pretranspile && yarn transpile",
-    "prettier": "prettier \"**/*.+(js|json|yml|css|scss|md|mdx)\"",
     "prettier:format": "yarn prettier --write",
+    "prettier": "prettier \"**/*.+(js|json|yml|css|scss|md|mdx)\"",
     "registrymock:publish": "lerna publish --no-git-tag-version --no-push --registry http://localhost:4873",
     "registrymock:start": "verdaccio",
+    "registrymock": "run-p -l registrymock:start registrymock:publish",
     "storybook:build": "build-storybook -c .storybook -o storybook-dist",
+    "storybook:link": "node scripts/storybookLink.js",
```

Comment:
I think since this is not generally invoked locally by developers, maybe we shouldn't have a script here to avoid confusion? and just have semaphore run node directly